### DISCUSSION TITLE
How do we add Metamask to desktopWallets?

### DISCUSSION THREAD

**Discussion by bores32** [WalletConnect/walletconnect-monorepo#3661] [Developer Support] [answered]
The following does not work:     

     desktopWallets: [
            {
              id: "metamask",
              name: "Metamask",
              links: {
                native: "metamask://",
                universal: "https://metamask.io",
              },
            }
          ]

WalletConnectModal, latest version.

**Comment by glitch-txs** [comment, marked as answer]
Hi WalletConnectModal doesn't work with browser extension wallets, however this feature is supported by [Web3Modal SDK](https://docs.walletconnect.com/2.0/web3modal/about)
